docs/source/topics/calculations/base.md

```diff
@@ -34,4 +34,4 @@ LAMMPS can produce binary restart files which contain all the atomic positions,
 - **structure**, ({class}`~aiida.orm.nodes.data.structure.StructureData`), *optional* - The output structure of the calculation.
 - **remote_folder**, ({class}`~aiida.orm.nodes.data.remote.base.RemoteData`) - Folder in the remote machine where the calculation was performed.
 - **remote_stash**, ({class}`~aiida.orm.nodes.data.remote.stash.base.RemoteStashData`), *optional* – Contents of the stash.source_list option are stored in this remote folder after job completion.
-- **retrieved**, ({class}`~aiida.orm.nodes.data.folder.FolderData`) - Files that are retrieved by the daemon will be stored in this node. By default the stdout and stderr of the scheduler will be added, but one can add more by specifying them in `CalcInfo.retrieve_list`.
+- **retrieved**, ({class}`~aiida.orm.nodes.data.folder.FolderData`) - Files that are retrieved by the daemon will be stored in this node. By default the stdout and stderr of the scheduler will be added, but one can add more by specifying them in `settings["additional_retrieve_list"] = ["foo", "bar"]`.
```

Comment:
Instead of explaining this option in the `settings` here, wouldn't it make more sense to actually add this to the explanation of the `settings` input itself? You also may want to add that wildcards are supported, i.e. you could add `*.xml` to retrieve all XML files for example

Reply:
Hi @sphuber ! Thanks for the comments, I have fixed them now, and added the explanation about wildcards with a link to the place in the aiida-core documentation where this is explained.